src/mux.js

```diff
@@ -52,6 +58,8 @@ class Mux extends Base {
 
     /** @type {Data} */
     this.Data = new Data(this);
+
+    this.Webhooks = new Webhooks(this);
```

Comment:
Since this doesn't need any of the prototype inheritance, let's just assign it as a static method like `JWT` for now.

Reply:
In your opinion, should webhooks move to `utils/` to keep it in `webhooks/webhooks`. I kinda like that latter but don't feel strongly